cssom-view-1/Overview.bs

```diff
@@ -1929,6 +1932,14 @@ Whenever a <a>visual viewport</a> gets scrolled (whether in response to user int
 
 When asked to <dfn export for=Document>run the scroll steps</dfn> for a {{Document}} <var>doc</var>, run these steps:
 
+1. For each item <var>target</var> in <var>doc</var>'s
+    <a spec="css-scroll-snap-2" lt="pending snapchanging event targets">pending snapchanging event targets</a>:
+
+    1. Fire a {{SnapEvent}}, |snapevent|, named <a spec=css-scroll-snap-2 event>snapchanging</a> at <var>target</var> and let |snapevent|'s {{SnapEvent/snapTargetBlock}} and
+        {{SnapEvent/snapTargetInline}} attributes be the <a spec=css-scroll-snap-2 lt="snapchangingTargetBlock">snapchangingTargetBlock</a> and the
+        <a spec=css-scroll-snap-2 lt="snapchangingTargetInline">snapchangingTargetInline</a>, respectively, that are
+        associated with <var>target</var>.
+1. Empty <var>doc</var>'s <a spec="css-scroll-snap-2" lt="pending snapchanging event targets">pending snapchanging event targets</a>.
```

Comment:
These steps should be moved to an algorithm in css-scroll-snap-2 as well, invoked from here.

Reply:
Good suggestion. Done.